src/include/duckdb/main/query_profiler.hpp

```diff
@@ -131,8 +132,8 @@ class QueryProfiler {
 	struct TreeNode {
 		PhysicalOperatorType type;
 		string name;
-		string extra_info;
-		OperatorInformation info;
+		TreeNodeSettings settings;
+		vector<unique_ptr<ExpressionExecutorInfo>> executors_info;
```

Comment:
Are these the child nodes? So, they are owned by the parent node?

Reply:
Yeah the TreeNode is a recursive struct so each layer has the settings.  Is that what you mean?